test/index.js

```diff
@@ -63,11 +63,9 @@ testEncodeDecode({ 1: true })
 
 tape('seekPath', (t) => {
   const path = ['dependencies', 'varint']
-  const pathBuf = Buffer.alloc(bipf.encodingLength(path))
-  bipf.encode(path, pathBuf, 0)
+  const pathBuf = bipf.allocAndEncode(path)
```

Comment:
yeah I was looking at that function in one of the other PRs and thought it would be quite helpful :)